streams/src/test/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignorTest.java

```diff
@@ -655,31 +669,33 @@ public void testAssignWithStandbyReplicas() {
 
     @Test
     public void testOnAssignment() {
-        configurePartitionAssignor(Collections.emptyMap());
+        createMockTaskManager();
+        final Map<HostInfo, Set<TopicPartition>> hostState = Collections.singletonMap(
+            new HostInfo("localhost", 9090),
+            Utils.mkSet(t3p0, t3p3));
+        taskManager.setPartitionsByHostState(hostState);
+        EasyMock.expectLastCall();
 
-        final List<TaskId> activeTaskList = asList(task0, task3);
```

Comment:
This stuff is just reorganized so that each field is defined just before it's used (so you can better tell how each field is used in the test).